cmd/syft/cli/options/attest.go

```diff
@@ -0,0 +1,28 @@
+package options
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type AttestOptions struct {
+	Key string
+}
+
+var _ Interface = (*AttestOptions)(nil)
+
+func (o *AttestOptions) AddFlags(cmd *cobra.Command, v *viper.Viper) error {
+	cmd.PersistentFlags().StringVarP(&o.Key, "key", "", "cosign.key",
+		"path to the private key file to use for attestation")
+
+	return bindAttestationConfigOptions(cmd.PersistentFlags(), v)
+}
+
+func bindAttestationConfigOptions(flags *pflag.FlagSet, v *viper.Viper) error {
```

Comment:
Todo: I think the packages command `password` for anchore enterprise conflicts with `password` for the attest command options. I need to check and see which overrides the new decorator's behavior and find a solution to distinguish between the two.

Reply:
Curious what the resolution was for this 😃

Reply:
This is a bug that exists on main at the moment. I have not found a good solution here besides renaming the flags. Sorry for resolving this comment that was an oversight on my part.

Reply:
Sorry @luhring I misspoke here. Attest does NOT provide a flag for password and there is no conflict. Passwords can only be set by ENV variable, passed in via the TUI, or set in the config.

We do not allow for plaintext on the CLI.